Tests/MongoSwiftTests/BsonValueTests.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+@testable import MongoSwift
+import Nimble
+import XCTest
+
+final class BsonValueTests: XCTestCase {
```

Comment:
Per my other comment, I think it'd ultimately make sense to split these by BSON type class when test coverage is expanded. I realize most of the successful inits for each type are probably covered in the Document tests.